### ISSUE TITLE
hide the pointer for a short time after writing

### ISSUE THREAD

**Issue by mrwth** [bordaigorl/rmview#18]
When I'm writing, the red pointer shows up immediately after the pen tip stops touching the screen, and so it shows up during writing (unless it's an uninterrupted line). It would be nice to have an option for showing the pointer only after some predefined short time since the last tip touch.

**Comment by ldecicco** [comment]
Just pushed a PR that implements this feature.

**Comment by bordaigorl** [comment]
@ldecicco is the `show_pen_on_lift` setting a good enough fix for this?

**Comment by ldecicco** [comment]
yes it is! thanks and sorry that I didn't provide my feedback sooner.
